include/crypto/x509.h

```diff
@@ -112,6 +112,9 @@ struct X509_crl_st {
     const X509_CRL_METHOD *meth;
     void *meth_data;
     CRYPTO_RWLOCK *lock;
+
+    OPENSSL_CTX *libctx;
+    const char *propq;
```

Comment:
I will have to fix the propq in a seperate pass... They all need to be duped not shallow copies.

Reply:
> They all need to be duped not shallow copies.

Memory is cheap!  [ahem]